packages/react/src/components/Search/Search.js

```diff
@@ -162,9 +162,12 @@ export default class Search extends Component {
     const CloseIconX = size === 'xl' ? Close20 : Close16;
 
     return (
-      <div className={searchClasses}>
+      <div
+        role="search"
```

Comment:
Should this always be on the `<Search>` component? I'm just not sure what the impact is 🤔 

It seems like `role="search"` is treated as a landmark and if folks are using this frequently it might clutter the list of regions on the page if we default here versus over in `TableToolbarSearch` we should have ideally one version of itself on thepage.

Reply:
From what I read, it seems like even if there are multiple searches on a page, each one still should be a landmark with a unique label of what it is for (data table, the whole site, etc). Since we didn't make the input type "search", the Search component isn't a landmark right now and probably should be without users needing to add it.